### ISSUE TITLE
ERROR:  Could not find a valid gem 'money-tree' (~> 0.11) (required by 'eth' (>= 0)) in any repository

### ISSUE THREAD

**Issue by marek22k** [se3000/ruby-eth#76]
Hello,

I can no longer update the eth gem or install it on another computer. This is very unfavorable as there are also dependencies on the gem.

Following is the error message:
```
$gem install eth
ERROR:  Could not find a valid gem 'money-tree' (~> 0.11) (required by 'eth' (>= 0)) in any repository
ERROR:  Possible alternatives: money-tree
```

```
$gem --version
3.3.5
```
```
$ruby -v
ruby 3.1.0p0 (2021-12-25 revision fb4df44d16) [x86_64-linux]
```
I installed Ruby using RVM.

**Comment by Deekor** [comment]
I'm also getting this trying to update to `eth 0.4.18`

**Comment by Deekor** [comment]
@marek22k Looks like version `0.11` of money-tree is not on ruby gems yet. Installing money-tree locally is a workaround.

`gem 'money-tree', git: 'https://github.com/GemHQ/money-tree.git'`

**Comment by q9f** [comment]
Indeed, there is not much we can do, ref https://github.com/GemHQ/money-tree/issues/50

Your options are:
1. either manually install money tree locally from git
2. or use the new (unreleased) eth gem 0.5.0: https://github.com/q9f/eth.rb/

**Comment by q9f** [comment]
Fixed in `~> 0.5`
